api/v1alpha1/status.go

```diff
@@ -130,6 +130,19 @@ type RuntimeStatus struct {
 	Mounts []Mount `json:"mounts,omitempty"`
 }
 
+// OperationStatus defines the observed state of certain Data Operation
+type OperationStatus struct {
+	// Phase describes current phase of DataBackup
```

Comment:
comments are not correct and results in the description of CRD not correct.

Reply:
Fixed. Thx.